man/erf.1

```diff
@@ -24,15 +24,17 @@ This tool supports version V1.0, as used by
 and
 .Em The Witcher
 and version V2.0 as used by
-.Em Dragon Age: Origins .
+.Em Dragon Age: Origins
+and version V2.2 as used by
+.Em Dragon Age: Origins
+and
+.Em Dragon Age 2
```

Comment:
Since you're changing the man page, please also update its date at the top of the file